### ISSUE TITLE
FEATURE REQUEST: Bulk editing in the Django admin control panel

### ISSUE THREAD

**Issue by burnbrighter** [meeb/tubesync#493]
Hi there - it doesn't seem like it would be hard to do  - perhaps you could start initially with key features, but would it be possible to add bulk edit in for at the Django admin interface for some key functions on some objects, such as Media?  A real time saver would be to expose actions such as enabling/disabling Skip or Can download.  This would be extremely helpful when editing a large number of objects instead of having to do each individually through the interface.   Thank you for your work on this.

![image](https://github.com/meeb/tubesync/assets/5279205/9b5c586d-ca59-42d1-8742-61280261c40c)
![image](https://github.com/meeb/tubesync/assets/5279205/e4c1fe4e-10c3-467f-ae23-ef98ac471930)

**Comment by meeb** [comment]
Yeah that sounds like a quick thing to add, I'll pop it on the top of the wishlist. Thanks!

**Comment by burnbrigther** [comment]
Thanks!

From: meeb ***@***.***>
Date: Thursday, April 18, 2024 at 7:43 PM
To: meeb/tubesync ***@***.***>
Cc: burnbrigther ***@***.***>, Author ***@***.***>
Subject: Re: [meeb/tubesync] FEATURE REQUEST: Bulk editing in the Django admin control panel (Issue #493)

Yeah that sounds like a quick thing to add, I'll pop it on the top of the wishlist. Thanks!

**Comment by tcely** [comment]
@meeb Please assign this to the `v1.0` milestone when you have a moment. Thanks.
